### DISCUSSION TITLE
yaml indentation

### DISCUSSION THREAD

**Discussion by IXLLEGACYIXL** [stride3d/stride#2129] [Ideas]
Currently the yamlserializer uses an identation of 4 which makes it more unreadable the deeper you go 

it wouldnt break anything existing, but it would affect the git history of all projects but the files would be more readable
also it would be compliant with "normal default" yaml indentation

for the parser it doesnt matter how much spaces there are afaik ( it does matter )

https://gist.github.com/IXLLEGACYIXL/1b906534da9e3c5d1cb450206943b3c9
as comparison, one is the old format with 4 spaces indentation and below with 2 spaces

**Comment by Kryptos-FR** [comment]
I could be set as a setting on the project. Then we can have an upgrader that will by default set it to 4 for backward compatibility, while new projects will have 2 by default. Changing the setting would force resaving all files but that's ok since that would be a manual operation by the user.
